api/test/model_device.py

```diff
@@ -92,3 +96,12 @@ def test_DeviceRemovedCorrectly(self):
     def test_DevicesCannotHaveSameName(self):
         with self.assertRaisesRegex(Exception, "There is already a device with this name."):
             self.devices.add_device(self.house1id, None, "Kitchen Thermostat", "thermostat", 1, None, "example")
+
+    def test_EnergenieDeviceAddedCorrectly(self):
+        device = self.devices.get_device_by_id(self.socket_id)
+        self.assertEqual(device['configuration']['device_id'], '46865', 'incorrect inserted device id')
+
+    def test_EnergenieDeviceReadState(self):
+        socket = self.devices.get_device_by_id(self.socket_id)
+        current_state = socket.read_current_state()
+        self.assertEquals(current_state['data']['data']['device_type'], 'control', 'state not read correctly')
```

Comment:
Thanks for the explanation. Please add this to the code if it is important to know!